### ISSUE TITLE
Does not work because of deprecated gvfs-info

### ISSUE THREAD

**Issue by Lalarian** [WinEunuuchs2Unix/iconic#2]
Hello!

I just tried the script, for making th desktop icons on my XFCE setup with three monitors more reliable.
I had to install some dependencies, but then I could not go on, because Xubuntu doesn't support gvfs-info anymore, instead it offers gio.

Could you update the script for using gio instead?

Thanks! :)

**Comment by WinEunuuchs2Unix** [comment]
GIO is certainly coming down the pipes but it will be awhile before that happens and odds are iconic will be rewritten in Python.

**Comment by Julius-Nepos** [comment]
So does this mean that Iconic can never work with the XFCE Desktop.  Oh by the way, I tried to submit a question here on installing the program but it seems to have disappeared.

Running an Ubuntu 18.04 64-bit system (with XFCE: which makes it Xubuntu) I can download this archive which has a text file.  How do I compile that?  Just marking it as an executable doesn't work.

**Comment by Lalarian** [comment]
"iconic" is a bash script.
Maybe you need to adapt some things inside the script like "ICONS_DIR="/home/$USER/Desktop"" as the name 'Desktop' changes with language. But because of the gio thing it won't work for Xubuntu anyways.

**Comment by evazzoler** [comment]
Please, fix it... It seems to be a very good piece of software!

**Comment by Julius-Nepos** [comment]
Yes I agree, seems to be very useful.  However, I'm not going to get rid of Xunbuntu and go back to Gnome just to run it.
